### PR TITLE
Update ptBR.lua

### PR DESCRIPTION
**Pull request by david95alex**

### CODE REVIEW

File: Nightmare/Locales/ptBR.lua [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+	L.yourLink = "Você esta linkado com %s"
+	L.yourLinkShort = "Linkado com %s"
 end
 
 L = BigWigs:NewBossLocale("Il'gynoth", "ptBR")
 if L then
-	--L.custom_off_deathglare_marker = "Deathglare Tentacle marker"
-	--L.custom_off_deathglare_marker_desc = "Mark Deathglare Tentacles with {rt6}{rt5}{rt4}{rt3}, requires promoted or leader.\n|cFFFF0000Only 1 person in the raid should have this enabled to prevent marking conflicts.|r\n|cFFADFF2FTIP: If the raid has chosen you to turn this on, having nameplates enabled or quickly mousing over the spears is the fastest way to mark them.|r"
+	L.custom_off_deathglare_marker = "Marcador de Tentáculo Fulgor da Morte"
+	--read all this line --L.custom_off_deathglare_marker_desc = "Marca Tentáculo FUlgor da Morte com {rt6}{rt5}{rt4}{rt3}, requer assistente ou líder.\n|cFFFF0000Apenas 1 pessoa no raide deve ter isto ativado para prevenir conflitos na marcação.|r\n|cFFADFF2FDICA: Se o raide escolheu você para isso, ter placas de identificação ativadas ou passar rapidamente o mouse sobre *******"the spears"(you mean the body of the tentacle?, if yes then you can use "as cabeças")**** é a maneira mais rápida de marcá-las.|r"
```
Comment (funkydude) [inline Nightmare/Locales/ptBR.lua]: It should say tentacles, the English version is wrong. You also have a small typo: "FUlgor"

File: Nightmare/Locales/ptBR.lua [hunk 2]
```diff
@@ -17,7 +17,7 @@ if L then
 	L.custom_off_deathglare_marker = "Marcador de Tentáculo Fulgor da Morte"
 	L.custom_off_deathglare_marker_desc = "Marca Tentáculo Fulgor da Morte com {rt6}{rt5}{rt4}{rt3}, requer assistente ou líder.\n|cFFFF0000Apenas 1 pessoa no raide deve ter isto ativado para prevenir conflitos na marcação.|r\n|cFFADFF2FDICA: Se o raide escolheu você para isso, ter placas de identificação ativadas ou passar rapidamente o mouse sobre os tentáculos é a maneira mais rápida de marcá-los.|r"
 
- 	L.bloods_remaining = "%d |4Sangue:Sangues; restando"
+ 	L.bloods_remaining = "%d |4Sangue:Sangues; faltando"
```
Comment (david95alex) [inline Nightmare/Locales/ptBR.lua]: This makes more sense for brazilian portuguese in what it means for the fight.

File: Nightmare/Locales/ptBR.lua [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
+	L.befouler = "Conspurcador Cordismáculo"
+	L.shaman = "Xamã Atroz"
 end
 
 L = BigWigs:NewBossLocale("Ursoc", "ptBR")
 if L then
-	--L.custom_off_gaze_assist = "Focused Gaze Assist"
-	--L.custom_off_gaze_assist_desc = "Show raid icons in bars and messages for Focused Gaze. Using {rt4} for odd, {rt6} for even soaks. Requires promoted or leader."
+	L.custom_off_gaze_assist = "Assistente de Olhar Concentrado"
+	L.custom_off_gaze_assist_desc = "Mostra marcadores de alvo nas barras e mensagens para Olhar Concentrado. Marca o alvo com {rt4} em casts ímpares e {rt6} em casts pares. Requer assistente ou líder.
```
Comment (funkydude) [inline Nightmare/Locales/ptBR.lua]: Build fails due to missing `"` at the end.

File: Nightmare/Locales/ptBR.lua [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
+	L.befouler = "Conspurcador Cordismáculo"
+	L.shaman = "Xamã Atroz"
 end
 
 L = BigWigs:NewBossLocale("Ursoc", "ptBR")
 if L then
-	--L.custom_off_gaze_assist = "Focused Gaze Assist"
-	--L.custom_off_gaze_assist_desc = "Show raid icons in bars and messages for Focused Gaze. Using {rt4} for odd, {rt6} for even soaks. Requires promoted or leader."
+	L.custom_off_gaze_assist = "Assistente de Olhar Concentrado"
+	L.custom_off_gaze_assist_desc = "Mostra marcadores de alvo nas barras e mensagens para Olhar Concentrado. Marca o alvo com {rt4} em casts ímpares e {rt6} em casts pares. Requer assistente ou líder."
```
Comment (funkydude) [inline Nightmare/Locales/ptBR.lua]: This translation is wrong, we don't mark players with rt4/6.

Comment (david95alex) [inline Nightmare/Locales/ptBR.lua]: No? What it does then?? Like i said, i did a LFR, i enabled this function in the option and i was raid assistant. The countdown bars for this boss skill had rt4/6, and when he casts it the target was indeed marked with the corresponding icon.

Comment (david95alex) [inline Nightmare/Locales/ptBR.lua]: Who was marking them with exactly rt4/6 then? I even saw this conflict with DBM, sometimes the skull icon remained on target but sometimes the rt4 or 6 remained, i thought it was BW

Comment (david95alex) [inline Nightmare/Locales/ptBR.lua]: i mean, i thought it was DBM marking skull and BW with rt4/6

**Pull request merged**